docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -4,64 +4,109 @@ title: "Red Hat OpenShift"
+Please note that any version not explicitly listed in the table above hasn't been tested, and we cannot guarantee compatibility.
 
-#### Permissive SCCs
+## Deploying Camunda 8 in OpenShift
 
-Out of the box, if you deploy Camunda 8 (and related infrastructure) with a permissive SCCs, there is nothing particular for you to configure. Here, a permissive SCCs refers to one where the strategy for `RunAsUser` is defined as `RunAsAny` (including root).
+Depending on your OpenShift cluster's Security Context Constraints (SCCs) configuration, the deployment process may vary.
 
-#### Non-root SCCs
+<Tabs>
+  <TabItem value="w-scc" label="Security Context Constraints (SCCs)" default>
 
-If you wish to deploy Camunda 8 but restrict the applications from running as root (e.g. the `nonroot` built-in SCCs), you will need to configure each pod and container to run as a non-root user. For example, when deploying Zeebe using a stateful set, you would add the following, replacing `1000` with the user ID you wish to use:
+### With restrictive SCCs
 
-```yaml
-spec:
-  template:
-    spec:
-      securityContext:
-        runAsUser: 1000
-      containers:
-        securityContext:
-          runAsUser: 1000
+By default, OpenShift employs more restrictive SCCs. The Helm chart must assign `null` to the user running all components and dependencies. Due to a [null bug](https://github.com/helm/helm/issues/9136) [in Helm](https://github.com/helm/helm/issues/12490), this operation must be executed using [a post-renderer](https://helm.sh/docs/topics/advanced/#post-rendering).
+
+To deploy Camunda 8 on OpenShift, please follow these installation steps:
+
+1. Install [Helm and other CLI tools](/self-managed/setup/install.md#prerequisites).
+2. Ensure that `bash` and `sed` are available locally, as they are necessary for the [post-rendering process to patch the values of OpenShift](https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/openshift/patch.sh).
+3. Install the [Camunda Helm chart repository](/self-managed/setup/install.md#helm-repository).
+4. Download the exact version of the chart that you want to install and untar it in a directory ([Camunda 8 Helm Chart Version Matrix](https://helm.camunda.io/camunda-platform/version-matrix/)):
+
+```shell
+# List of available version: https://helm.camunda.io/camunda-platform/version-matrix/
+export CHART_VERSION="pleaseDefine"
+
+# Make sure to set CHART_VERSION to match the chart version you want to install.
+helm pull camunda/camunda-platform --version "$CHART_VERSION" --untar --untardir "/tmp/camunda-platform-$CHART_VERSION"
 ```
 
-:::note
-As the container user in OpenShift is always part of the root group, it's not necessary to define a `fsGroup` for any Camunda 8 applications pod security context.
+5. Configure the chart values:
+
+```shell
+cp "/tmp/camunda-platform-$CHART_VERSION/camunda-platform/values.yaml"  "/tmp/camunda-platform-$CHART_VERSION/camunda-platform/my-values.yaml"
```

Comment:
the way it seems, we're recommending to copy over the default values and adjust them - but why? it's usually pretty hard to debug, and usually we should aim for the users to have as lean custom values as possible.